templates/repo/release/list.tmpl

```diff
@@ -75,6 +75,7 @@
 				</li>
 			{{end}}
 		</ul>
+		{{template "admin/base/page" .}}
```

Comment:
We can copy admin/base/page to base/page and fix here since user's pager maybe not the same style with admin pager.